### PR TITLE
Lesson 04 jupyter notebook

### PR DESCRIPTION
**Pull request by dongyoungkim-dy**
# We explored the use of dictionary
## It was fun

### CODE REVIEW

**Comment by jvafable** [comment]
I also had fun

File: dy_lesson_04/lesson4.ipynb [hunk 1]
```diff
@@ -0,0 +1,994 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "dict_items([('Bob', 10), ('Alice', 15), ('DY', 7)])\n",
```
Comment (jvafable) [inline dy_lesson_04/lesson4.ipynb:132]: This is an example of a comment in a pull request for a review

Comment (dongyoungkim-dy) [reply dy_lesson_04/lesson4.ipynb:132]: What should be fixed? Please elaborate more

Comment (jvafable) [reply dy_lesson_04/lesson4.ipynb:132]: this looks good

**Pull request merged**